src/preprocessing-service/preprocess.py

```diff
@@ -153,13 +163,21 @@ async def es_log_management():
         await asyncio.sleep(3600)  # deletion once an hour
 
 
+async def init_nats():
```

Comment:
I noticed that this function appears everywhere, is there a better way to do this?

Reply:
unfortunately we need to initialize nats before passing its instance off to the other coroutines. let me know if you have any suggestions

Reply:
it's fine, let's not worry about this for now then.